README.md

```diff
@@ -80,6 +80,18 @@ $ LDFLAGS=-L/usr/local/opt/openssl/lib \
   ./build.sh
 ```
 
+If the Gambit interpreter has a nonstandard name or installation path:
```

Comment:
I would add a bit more explanation here: "Gerbil expects the gambit interpreter to be called gsi; if for some reason gsi has a nonstandard name (eg. in Arch Linux gsi is the ghostscript interpreter), then ..."